### PR TITLE
Responsive Web Design created

### PR DESCRIPTION
**Pull request by vvoo21**
Hi 😃, 

I created my Portfolio: desktop version adaptable to different screen sizes using media queries. 

I made all this following the instructions of Portfolio: desktop version:
- Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider).
- In order to lay out the elements on the page you should use Grid and Flexbox.

- You don't need to implement any functionality that requires JavaScript.
 

Happy day/night!!

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -344,7 +360,6 @@ form {
 .contact-form {
   display: flex;
   flex-direction: column;
-  justify-content: flex-start;
   gap: 21px;
 }
```
Comment (ha-manel) [inline style.css]: - [ ] Kindly update the form inputs to make them match Figma (`input width`, `font size`, `border radius`, and `padding`)

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @vvoo21 👋 
**Your website looks awesome, great styling skills 👏💯** 
There is a tiny change to make before you can proceed to the next project 😄



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Please check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vvoo21** [comment]
Hi @ha-manel,
I have made the requested changes👩‍💻. 
Thank you.

**Review by mgmediaweb — APPROVED** [review]
Hi @vvoo21 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved5

## Highlights 💯 
- Good job following the advice from the before reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vvoo21** [comment]
Hi @mgmediaweb,
thank you so much 😃!

**Pull request merged**